### DISCUSSION TITLE
Searching for messages beyond 90 days

### DISCUSSION THREAD

**Discussion by losuler** [convos-chat/convos#550] [Q&A] [answered]
A distinguishing feature of Convos compared to similar clients I've tried is the search feature, however I noticed it says on the search page that it's limited to messages in the last 90 days.

Is there a reason for this limitation (e.g. performance) and would it be possible to increase it if I were to accept aspects such as increased search time?

**Comment by jhthorsen** [comment, marked as answer]
Yes. It's mostly for performance, though 90 days is just a guess. You can increase it by setting `CONVOS_DEFAULT_SEARCH_AFTER`. Example:

```
# 31536000 = 365 * 86400 # 365 days * seconds_per_day
CONVOS_DEFAULT_SEARCH_AFTER=31536000 ./script/convos daemon
```

Unfortunately, the text on the search page will not update, since it's a fixed value: https://github.com/Nordaaker/convos/blob/296d286aaff60ded33bc2e792ea7bd1e9caef56c/assets/page/Search.svelte#L67
